src/test/data/JsonSerializableWorkBookTest/invalidInternshipWorkBook.json

```diff
@@ -2,9 +2,7 @@
   "internships": [ {
     "company": "Hans Muster",
     "role": "Software Engineer",
-    "phone": "9482424",
     "email": "invalid@email!3e",
-    "address": "4th street",
```

Comment:
Any reason to remove this address attribute? Lmk!

Reply:
I removed it because we are not using an address attribute anymore. On hindsight this change should have been made in a separate bug fixing commit